Topic_2/blog/src/test/java/com/mycompany/app/blog/BlogTest.java

```diff
@@ -0,0 +1,70 @@
+package com.mycompany.app.blog;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class BlogTest {
+	private Blog blog;
+	private int id;
+	
+	@Before
+	public void setUp() {
+		blog = new Blog();		
+	}	
+
+	@Test
+	public void whenANewEntryIsSentToTheBlogThenItIsAddedToTheListOfEntries() {		
+		id = blog.newEntry("title1 test1","body");
+		assertNotNull(blog.searchEntry(id));		
+	}
+
+	@Test
+	public void whenAnEntryIsDeletedThenItIsRemovedFromTheListOfEntries() {	
+		blog.newEntry("title1 test2","body");
+		id = blog.newEntry("title2 test2","body");	
+		blog.newEntry("title3 test2","body");	
+		assertNotNull(blog.searchEntry(id));		
+		
+		blog.deleteEntry(id);
+		assertNull(blog.searchEntry(id));
+	}
+	
+	@Test
+	public void returnsThe10mostRecentEntries() {
```

Comment:
This test is wrong. You need to verify if the list of most recent entries is the same that get from getMostRecentEntries().